testdata/expected/program/python-single-module-lib/single-module-lib/PipPackage.graph.json

```diff
@@ -1,104 +0,0 @@
-{
```

Comment:
There is a clash: docker's variant was removed, program variant was moved and updated, however git displays that:

- program's variant was removed (https://github.com/sourcegraph/srclib-python/pull/33/files#diff-65c9caf4a94b35ed16ae602a0703874e)
- docker's variant was moved and updated (https://github.com/sourcegraph/srclib-python/pull/33/files#diff-4ec9a286de93d4eabf0a4660c8350761)